tasks/SticklerForRules.tex

```diff
@@ -0,0 +1,46 @@
+\section{Stickler for the Rules [Party Host]}
+The robot has to enforce house rules set by the homeowner.
+
+
+\subsection{Main Goal}
+The robot has to recognize when party guests are breaking rules and politely ask them to stop.
+
+\noindent\textbf{Reward:} 500pts (100pts per recognized rule breaking).
+
+\subsection{Setup}
+\begin{itemize}[nosep]
+	\item \textbf{Location:} This test takes place in the arena.
+	\item \textbf{Guests:} There are several party guests inside the arena.
+\end{itemize}
+
+
+\subsection{Additional rules and remarks}
+\textbf{House Rules:}
+\begin{enumerate}[nosep]
+	\item No shoes allowed inside the house. All guests have to take off their shoes near the entrance.
+	\item No guests are allowed to enter one of the rooms in the house.
+	\item Guests are not allowed to drop garbage on the floor.
+	\item All guests have to have a drink in hand at all times.
+\end{enumerate}
+
+\subsection{Referee instructions}
+
+The referee needs to
+\begin{itemize}
+	\item Instruct party guests on which rules to break.
+\end{itemize}
```

Comment:
You're missing:
- Assign/provide drinks to each guest (that can be done by OC)
- If the robot needs to recognize people, names must be assigned

Reply:
no recognizing people needed I think